docs/getting_started/install.md

```diff
@@ -7,25 +7,48 @@ GluonTS is available from PyPi via:
 pip install gluonts
 ````
 
-```{attention}
-**GluonTS uses a minimal dependency model.**
+```{caution}
 
-This means that to use most models and features additional dependencies need to
-be installed. See the next section for more information.
+GluonTS uses [Semantic Versioning](https://semver.org) for managing versions.
+Since the library is actively developed we use `v0` as the major version.
+
+We plan to release a new minor version at the end of each quarter. The current
+minor release is `0.10`, with `0.11` being scheduled to be released end of
+September 2022.
```

Comment:
I think this information should not go there: maybe we can link to the milestones on GitHub instead?

Reply:
Changed it.